docker-compose.yml

```diff
@@ -90,6 +90,37 @@ services:
     profiles:
       - prover
 
+  prometheus:
```

Comment:
If this is for monitoring validator, do we need to add `depend_on` validator and node?